.travis.yml

```diff
@@ -7,18 +7,17 @@ addons:
 
 language: node_js
 node_js:
-- '0.10' # these should be quoted; otherwise, 0.10 turns into 0.1
-- '0.12'
-- '4'
+- 'node' # these should be quoted; otherwise, 0.10 turns into 0.1
```

Comment:
Out of curiosity, what version is `node`?

Reply:
`node`is current stable / latest which is currently 11.1.0.